types/three/src/Meta.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface Meta<Type extends string, Generator extends string, Version extends number = 4.5> {
+    version: Version;
+    type: Type;
+    generator: Generator;
+}
```

Comment:
I would really like to avoid creating a file that doesn't exist in three.js. Looks likes this is used in 5 places. Could we either just in-line it in the places it's used (and get rid of the interface altogether) or move it into a file that already exists like `Object3D.d.ts`?